### PR TITLE
Update packages to the latest versions

### PR DESCRIPTION
**Pull request by supertassu**
Unblock builds on the new infrastructure.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     }
   ],
   "dependencies": {
-    "grunt": "0.4.5",
-    "grunt-jquery-content": "3.0.0"
+    "grunt": "1.6.1",
+    "grunt-cli": "1.4.3",
```
Comment (Krinkle) [inline package.json:25]: grunt-cli should not be needed. Afaik we only invoke `grunt` via `npm run …` or `node_modules/.bin/`, which is already included in grunt.

If we need it globally outside npm, then the dependency wouldn't do that. In most repos, I've ended up removing this over the years. Looks like we forgot to remove it from api.jqueryui.com, and the state of this old branch suggests there was even a time where we didn't yet specify it even.

Long story short: Can we do it without this?

**Review by Krinkle — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -21,7 +21,8 @@
     }
   ],
   "dependencies": {
-    "grunt": "0.4.5",
-    "grunt-jquery-content": "3.0.0"
+    "grunt": "1.6.1",
```
Comment (Krinkle) [inline package.json:24]: 1.5.3 is the last that supports Node 10 (old infra), 1.6 requires Node 16. That's fine but means it'll break the old infra's staging and we shouldn't tag any stable versions on 1.8.x until we're transitioned.

Alternatively, if 1.5.3 works, maybe we can stay pinned to that on old branches. That should work fine on latest Node as well.

File: package.json (hunk 1 above)
Comment (supertassu) [reply package.json]: I'm not quite sure what I did wrong the first time I tested this, but yes, looks like this works correctly without an explicit `grunt-cli` require. Updated.

File: package.json (hunk 2 above)
Comment (supertassu) [reply package.json]: 1.5.3 works, done.
